python/pyspark/sql/avro/functions.py

```diff
@@ -0,0 +1,134 @@
+#
```

Comment:
BTW, `python/docs/pyspark.sql.rst` would have to be fixed so that it can be shown in PySpark API doc.

Reply:
Yeah, that's missing, addressing it...